specifyweb/frontend/js_src/lib/components/DataModel/businessRuleDefs.ts

```diff
@@ -308,8 +309,16 @@ export const businessRuleDefs: MappedBusinessRuleDefs = {
         cog.businessRuleManager?.checkField('cogType');
       }
     },
-    onRemoved(_, collection) {
+    onRemoved(cojo, collection) {
       // Trigger Consolidated COGs field check when a child is deleted
+      if (cojo.get('isPrimary')) {
+        setSaveBlockers(
+          collection.related!,
```

Comment:
```suggestion
          collection.related ?? cojo,
```
I don't expect `collection.related` to be undefined but a failsafe would be good to have